src/communication/network_interface_cl.c

```diff
@@ -10560,3 +10561,160 @@ loaddb_update_stats ()
   return NO_ERROR;
 #endif /* !CS_MODE */
 }
+
+int
+flashback_get_summary (dynamic_array * class_list, const char *user, time_t start_time, time_t end_time,
+		       void *summary_list, OID ** oid_list)
+{
+#if defined(CS_MODE)
+  int req_error = ER_FAILED;
+  int rep_error = ER_FAILED;
+  int request_size = 0;
+  char *request = NULL, *ptr, *start_ptr;
+  int num_tables = 0;
+
+  char table[SM_MAX_IDENTIFIER_LENGTH];
+
+  OR_ALIGNED_BUF (OR_INT_SIZE + OR_INT_SIZE) a_reply;
+  char *reply = OR_ALIGNED_BUF_START (a_reply);
+  char *area;
+  int area_size;
+
+  num_tables = da_size (class_list);
+
+  /* request : num tables | table name list | user | start_time | end_time */
+  request_size = or_packed_string_length (user, NULL) + OR_INT64_SIZE + OR_INT64_SIZE;
+
+  for (int i = 0; i < num_tables; i++)
+    {
+      if (da_get (class_list, i, table) != NO_ERROR)
+	{
+	  return ER_FAILED;
+	}
+      request_size += or_packed_string_length (table, NULL);
+    }
+
+  request = (char *) malloc (request_size + MAX_ALIGNMENT);
+  if (request == NULL)
+    {
+      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_OUT_OF_VIRTUAL_MEMORY, 1, (size_t) request_size);
+      return ER_FAILED;
```

Comment:
```suggestion
      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_OUT_OF_VIRTUAL_MEMORY, 1, (size_t) request_size);
      return ER_OUT_OF_VIRTUAL_MEMORY;
```
It doesn't need to match?

Reply:
OK, I will change it. Thanks